src/groups/mqb/mqbs/mqbs_filestoreprotocol.h

```diff
@@ -1160,10 +1162,20 @@ struct MessageRecord {
     static const int  k_CAT_START_IDX = 0;
     static const char k_CAT_MASK;
 
+    // PRIVATE CONSTANTS
```

Comment:
Please update the ascii illustration above, and comment that refCount is now split in two.